freezer.tex

```diff
@@ -151,11 +151,64 @@ \section{The Freezer}
 to avoid conflict. To do this, press \megakey{8} to toggle the unit assignment of drive 0
 between 8 and 10, and \megakey{9} to toggle the unit assignment of drive 1, between 9 and 11.
 
-After setting the preferences, the {\bf FREEZER} can be exited
-with \megakey{F3}. The {\bf FREEZER} restores the screen and
+After setting the preferences, the Freezer can be exited
+with \megakey{F3}. The Freezer restores the screen and
 returns to the interrupted program.
 
 Note: The drive and unit assignments are temporary, and will be reset to their defaults
 after power down or reset. For permanent settings, use the {\bf CONFIGURE}
 menu. You can read more about this menu on page \pageref{configuring-chipset}.
 
+\section{Using D81 images}
+As previously mentioned, the MEGA65 allows the use of D81 files instead of using physical disks. The MEGA65 includes a
+built-in file browser, which allows you to choose which D81 file to mount. To do this, enter the Freezer menu,
+then press \megakey{0} or \megakey{1}, to choose the drive to mount it to.
+
+When you do this, the file browser will appear:
+
+\begin{center}
+\includegraphics[trim= 10mm 20mm 10mm 20mm,clip,width=0.7\linewidth]{images/d81-file-browser.png}
+\end{center}
+
+The file browser will list all D81 images on the microSD card. The MEGA65 prioritises the microSD card, so if you
```

Comment:
Could we say "...prioritises the **external** microSD card...". Just that even after all this time with the devkit, when I hear someone microSD, I have to stop for a few seconds to try and recall, "Hmm, microSD... Is that the external one? Or the internal one?".